docs/sdk/client-side/android/index.md

```diff
@@ -158,6 +167,96 @@ viewLifecycleOwner.lifecycleScope.launch {
 </TabItem>
 </Tabs>
 
+#### Polling
+
+The initialize process immediately starts polling the latest evaluations from Bucketeer in the background using the interval `pollingInterval` configuration while the application is in the **foreground state**.
+When the application changes to the **background state**, it will use the `backgroundPollingInterval` configuration.
+
+#### Polling retry behavior
+
+The Bucketeer SDK regularly polls the latest evaluations from Buckteer based on the pollingInterval parameter. By default, the `pollingInterval` is set to 10 minutes, but you can adjust it to suit your needs.
+
+If a polling request fails, the SDK initiates a retry procedure. The SDK attempts a new polling request every minute up to 5 times. If all five retry attempts fail, the SDK sends a new polling request once the `pollingInterval` time elapses. The table below illustrates this scenario:
+
+<div className="center-table">
+<table>
+<thead>
+  <tr>
+    <th>Polling Time</th>
+    <th>Retry Time</th>
+    <th>Request Status</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>10:00</td>
+    <td>-</td>
+    <td>Fail</td>
+  </tr>
+  <tr>
+    <td>- </td>
+    <td>10:01</td>
+    <td>Fail</td>
+  </tr>
+  <tr>
+    <td>- </td>
+    <td>10:02</td>
+    <td>Fail</td>
+  </tr>
+  <tr>
+    <td>- </td>
+    <td>10:03</td>
+    <td>Fail</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>10:04</td>
+    <td>Fail</td>
+  </tr>
+  <tr>
+    <td>-</td>
+    <td>10:05</td>
+    <td>Fail</td>
+  </tr>
+  <tr>
+    <td>10:10</td>
+    <td>-</td>
+    <td>Successful</td>
+  </tr>
+</tbody>
+</table>
+</div>
+
+The polling counter, which uses the `pollingInterval` information, resets in case of a successful retry. The table below exemplifies the described scenario.
```

Comment:
```suggestion
The polling counter, which uses the `pollingInterval` information, resets in case of a successful retry. The table below shows the described scenario.
```

Reply:
Done